includes/wikia/services/RevisionUpvotesService.class.php

```diff
@@ -79,8 +79,8 @@ public function getRevisionUpvotes( $wikiId, $revisionId ) {
 
 		$upvote = ( new \WikiaSQL() )
 			->SELECT_ALL()
-			->FROM( self::UPVOTE_REVISIONS_TABLE )->AS_( 'revs' )
-			->INNER_JOIN( self::UPVOTE_TABLE )->AS_( 'uv' )
+			->FROM( static::UPVOTE_REVISIONS_TABLE )->AS_( 'revs' )
+			->INNER_JOIN( static::UPVOTE_TABLE )->AS_( 'uv' )
 			->ON( 'revs.upvote_id', 'uv.upvote_id' )
 			->WHERE( 'wiki_id' )->EQUAL_TO( $wikiId )
 			->AND_( 'revision_id' )->EQUAL_TO( $revisionId )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "revision_id" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)